CONTRIBUTING.md

```diff
@@ -120,8 +122,7 @@ visibility on the problem and work toward resolution.
 
 ## Closing issues
 
-You can either close issues manually by adding the fixing commit SHA1 to the issue
-comments or by adding the `Fixes` keyword to your commit message:
+You must add a `Fixes` keyword to one of your commits:
```

Comment:
If you look further down that example, you'll see `Fixes #121`. The title start with "Fix handling".

I've updated that example with an actual commit entry.